torch/utils/hipify/hipify_python.py

```diff
@@ -809,8 +809,9 @@ def repl(m):
     # Include header if device code is contained.
     output_source = hip_header_magic(output_source)
 
+    # NOTE: No longer needed; compiler now handles extern __shared__ correctly without special syntax.
     # Replace the extern __shared__
-    output_source = replace_extern_shared(output_source)
+    # output_source = replace_extern_shared(output_source)
```

Comment:
This hipify change could be breaking the internal builds, if the internal builds are based on older rocm versions.

Reply:
@jeffdaily Would you say this change is required for this PR, or is it an unrelated change?

Reply:
Shouldn't be needed.  But have we gotten word from @seemethere or @malfet if this was the source of the CI failure?

Reply:
```suggestion
    output_source = replace_extern_shared(output_source)
```